pylabrobot/resources/__init__.py

```diff
@@ -25,10 +25,13 @@
 
 # labware manufacturers and suppliers
 from .corning_costar import *
+from .corning_axygen import *
 from .revvity import *
 
 # liquid handling companies
 from .hamilton import HamiltonDeck, STARLetDeck, STARDeck
 from .ml_star import *
 from .opentrons import *
 from .tecan import *
+
+# labware made from 3rd parties that share their designs with PLR
```

Comment:
do you want this to be a part of this pr?

Reply:
Yes, just to get the DIY of labware and their PLR definitions sharing started :)